en/micro-integrator/docs/references/mediators/call-Template-Mediator.md

```diff
@@ -145,3 +152,39 @@ dynamically set via the message context.
         </sequence>
     </template>
 ```
+
+### Example 3
+
+Consider an example where the sequence template is configured to log the greeting message that is passed from the mediation sequence in the REST API. According to the sequence template, a value for the greeting message is mandatory. 
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<template name="sequence-temp" xmlns="http://ws.apache.org/ns/synapse">
+    <parameter isMandatory="false" defaultValue="Welcome" name="greeting_message"/>
```

Comment:
`a value for the greeting message is mandatory` but here isMandatory="false". Seems there is a conflict.

Reply:
Fixed